doc/man1/openssl.pod

```diff
@@ -545,6 +553,36 @@ Read the password from standard input.
 
 =back
 
+=head2 Random state options
+
+Prior to OpenSSL 3.0, it was common for applications to store information
+about the state of the random-number generator in a file that was loaded
+at startup and rewritten upon exit. On modern operating systems, this is
+generally no longer necessary as OpenSSL will seed itself from the
+appropriate CPU flags, device files, and so on. These flags are still
+supported for special platforms or circumstances that might require them.
+
+It is generally an error to use the same seed file more than once and
+every use of B<-rand> should be paired with B<-writerand>.
+
+=over 4
+
+=item B<-rand> I<file>
```

Comment:
May I suggest plural, i.e `I<files>`, to indicate that it may be several files? When I look more closely at this option, that turned out to be a better way to say it than the ellipses we currently have for this option all over the place.

Reply:
that's an excellent idea.  will do.